### DISCUSSION TITLE
Bug in checkbox

### DISCUSSION THREAD

**Discussion by ducnhoxlatui** [Budibase/budibase#10788] [Help] [answered]
Hi guys!
I have found that in that checkbox field, it's a bug or something like that. Does anybody meet this problem like me?

https://github.com/Budibase/budibase/assets/71976420/a121b6f4-d0c9-4a28-b348-5c694b975cdb

**Comment by andz-bb** [comment]
Hey @ducnhoxlatui, how is your checkbox field set up? Could you share an app export so that I can investigate? You can send it to me directly on [our Discord server](https://discord.gg/ZepTmGbtfF) (I'm `Andrew T#0001`) or to support@budibase.com - thanks!

**Reply by ConorWebb96** [reply]
Hey @ducnhoxlatui, 

I'm just following up on this,

This looks exactly like what would happen if you have different label names but the values of each checkbox is the same. Can you check the values used for these checkboxes? Additionally you can do what Andrew above has said and provide an app export for one of us to have a more in-depth look.

Thanks ahead of time!

**Reply by ducnhoxlatui** [reply]
Hi @ConorWebb96 , I repiled Andrew on Discord and provided him my app so that he can take a look in that, still wait for the answer now, LOL.

**Reply by andz-bb** [reply, marked as answer]
hey @ducnhoxlatui, I've replied to your message on Discord - hopefully this resolves your issue!

**Reply by vaishnaray** [reply]
@andz-bb I'm having the same issue, could you provide me the steps to resolve this issue?
